src/main/java/cloud/eppo/Utils.java

```diff
@@ -40,7 +40,7 @@ public static String getMD5Hex(String input) {
     return hashText.toString();
   }
 
-  public static Date parseUtcISODateElement(JsonNode isoDateStringElement) {
+  public static Date parseUtcISODateNode(JsonNode isoDateStringElement) {
```

Comment:
method name didn't change in our switch from org.json to Jackson, so updating to keep things parallel